help/destinations/catalog/mobile-engagement/line.md

```diff
@@ -0,0 +1,141 @@
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL LINE] Messaging API](https://developers.line.biz/en/reference/messaging-api/), add profiles to your segment and activate them as connections within [!DNL LINE] for your business needs.
+
+[!DNL LINE] uses Bearer Tokens as the authentication mechanism to communicate with the [[!DNL LINE] Messaging API]. Instructions to authenticate to your [!DNL LINE] instance are further below, within [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+As a marketer, I want to target users in a mobile engagement destination, with segments built in [!DNL Adobe Experience Platform]. Additionally, I want to deliver personalized experiences to them, based on attributes from their [!DNL Adobe Experience Platform] profiles, as soon as segments and profiles are updated in [!DNL Adobe Experience Platform].
+
+## Prerequisites {#prerequisites}
+
+### Experience Platform prerequisites {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL LINE] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+Read [Activate profiles and segments to streaming segment export destinations](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/activate/activate-segment-streaming-destinations.html?lang=en) for instructions on activating audience segments to this destination.
+
+### LINE prerequisites {#prerequisites-destination}
+
+Note the following prerequisites in [!DNL LINE], in order to export data from Platform to your [!DNL LINE] account:
+
+#### You need to have a LINE account {#prerequisites-account}
+
+Go to the [!DNL LINE] [signup](https://account.line.biz/signup) page to register and create an account, if you do not have one already.
+
+#### Gather the LINE channel access token (long-lived) from the LINE developer console {#gather-credentials}
+
+* To allow Platform to access [!DNL LINE] resources you will need the *[!DNL Channel access token (long-lived)]* from the desired [!DNL LINE] *Messaging API* channel.
+* Log in with your [!DNL LINE] account to the [[!DNL LINE] Developer console](https://developers.line.biz/console).
+* Next, access the *[!DNL Providers]* list, then select the *[!DNL Provider]* of interest and finally select the *Messaging API* channel to access its settings.
+    * If you are accessing the developer console for the first time please follow the [[!DNL LINE] documentation](https://developers.line.biz/en/docs/messaging-api/getting-started/) to complete the steps required to create a provider.
```

Comment:
You can just move this sentence in the same numbered list step as the sentence above, to read like: 

`Next, access the *[!DNL Providers]* list, then select the *[!DNL Provider]* of interest and finally select the *Messaging API* channel to access its settings. If you are accessing the developer console for the first time, please follow the [[!DNL LINE] documentation](https://developers.line.biz/en/docs/messaging-api/getting-started/) to complete the steps required to create a provider.`